ilastik/ilastik_logging/default_config.py

```diff
@@ -129,6 +156,15 @@ def get_default_config(
                 "formatter": "verbose",
                 "encoding": "utf-8",
             },
+            SESSION_FILEHANDLER_NAME: {
+                "level": "DEBUG",
+                "class": "logging.handlers.RotatingFileHandler",
+                "filename": SESSION_LOGFILE_PATH,
+                "maxBytes": 50e6,  # 20 MB
+                "backupCount": 5,
+                "formatter": "verbose",
+                "encoding": "utf-8",
+            },
```

Comment:
naive question: is the intention to keep all session logs?

Reply:
I was going to leave it for this PR and deal with it in the context of https://github.com/ilastik/ilastik/issues/2717 soon-ish

Reply:
I'm asking as I already have 10+ session logs...

Reply:
How many session logs do you reckon we should keep? 10?